### DISCUSSION TITLE
Is Extensions.Hosting only supported on UWP or WinUI

### DISCUSSION THREAD

**Discussion by oliverw** [unoplatform/uno#11151] [Q&A] [answered]
From reading [this Page](https://platform.uno/docs/articles/external/uno.extensions/doc/Learn/Tutorials/Hosting/HowTo-HostingSetup.html) I got the impression that the hosting extensions including DI are only supported when running on UWP or WinUI. So the app is no longer cross-platform? Am I missing something?

**Comment by jeromelaban** [comment, marked as answer]
When we're referring to UWP or WinUI in the Uno Platform documentation, we're referring to the API set, not the Windows implementation of the frameworks. Uno Platform implements both API sets, using nuget packages `Uno.UI` for UWP, and `Uno.WinUI` for WinUI 3/WinAppSDK.

The best way to start with Extensions.Hosting is to start with the [Uno.Extensions template](https://platform.uno/docs/articles/external/uno.extensions/doc/Overview/ExtensionsOverviewAndGettingStarted.html#step-by-steps).
